types/jss/jss-tests.ts

```diff
@@ -7,10 +7,17 @@ import {
 
 const jss = createJSS().setup({});
 
-const styleSheet = jss.createStyleSheet(
+const styleSheet = jss.createStyleSheet<string>(
 	{
 		ruleWithMockObservable: {
-			subscribe() {}
+			subscribe: observer => {
+				const next = typeof observer === 'function' ? observer : observer.next;
+				next({ background: 'blue', display: 'flex' });
+				next({ invalidKey: 'blueish' }); // $ExpectError
+				return {
+					unsubscribe() {}
+				};
```

Comment:
How does this test look?

Reply:
Does it pass?

You might have to call `createStyleSheet` once with the expected good one and again with the expected bad one, since it isn't an error to call `next` with arbitrary data (only to try to make a stylesheet from an observable that sends arbitrary data).

This might be a better test:

```typescript
declare const style$: Observable<{ background: 'blue', display: 'flex' }>;
declare const notStyle$: Observable<{ wrongKey: 'some value' }>;
// replace `ruleWithMockObservable` with `movingElement: style$`
jss.createStyleSheet<string>({ movingElement: notStyle$ }); //$ExpectError
```

Apologies for not pointing you this way earlier.